src/client/visualizations/bar-chart/bar-chart.tsx

```diff
@@ -518,15 +518,15 @@ export class BarChart extends BaseVisualization<BarChartState> {
 
         labels.push(<div
           className="slanty-label continuous"
-          key={segmentValueStr}
+          key={i}
```

Comment:
This should probably be `key={String(segmentValue)}` that would work correctly on all types of ranges and values